src/patternfly/patterns/SecondaryNav/styles.scss

```diff
@@ -1,17 +1,17 @@
 @import "../../patternfly-utilities";
 // pf-p-block__element--modifier--state--PropiertyCamel
-$pf-p-secondary-nav--BorderColor:                $pf-global--light--BorderColor !default;
-$pf-p-secondary-nav--BorderWidth:                $pf-global--BorderWidth !default;
+$pf-p-secondary-nav--BorderColor:                $global--BorderColor--light !default;
+$pf-p-secondary-nav--BorderWidth:                $global--BorderWidth !default;
 
-$pf-p-secondary-nav__item--MarginLeft:           $pf-global--spacer--md !default;
-$pf-p-secondary-nav__item--MarginRight:          $pf-global--spacer--md !default;
+$pf-p-secondary-nav__item--MarginLeft:           $global--spacer--lg !default;
+$pf-p-secondary-nav__item--MarginRight:          $global--spacer--lg !default;
 
-$pf-p-secondary-nav__link--Color:                $pf-global--dark-color !default;
-$pf-p-secondary-nav__link--PaddingTop:           $pf-global--spacer--sm !default;
-$pf-p-secondary-nav__link--PaddingBottom:        $pf-global--spacer--sm !default;
+$pf-p-secondary-nav__link--Color:                $global--Color--dark !default;
+$pf-p-secondary-nav__link--PaddingTop:           $global--spacer--md !default;
+$pf-p-secondary-nav__link--PaddingBottom:        $global--spacer--md !default;
 
-$pf-p-secondary-nav__link--active--Color:        $pf-global--active--Color !default;
-$pf-p-secondary-nav__link--active--BorderColor:  $pf-global--active--Color !default;
+$pf-p-secondary-nav__link--active--Color:        $global--Color--active !default;
+$pf-p-secondary-nav__link--active--BorderColor:  $global--Color--active !default;
```

Comment:
I think this one was probably meant to be something else. It's using a variable that is only meant to be used with the color property.